### PR TITLE
support bionic (18.04)

### PR DESCRIPTION
**Pull request by rogpeppe**

### CODE REVIEW

File: internal/charmstore/store.go [hunk 1]
```diff
@@ -797,7 +797,8 @@ var seriesScore = map[string]int{
 	"wily":    6,
 	"yakkety": 7,
 	"zesty":   8,
-	"artful":  0, // TODO(mhilton) make this higher when artful is released
+	"artful":  9,
+	"bionic":  0, // TODO(mhilton) make this higher when bionic is released
```
Comment (mhilton) [inline internal/charmstore/store.go]: hey, why are you giving me TODOs? :)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2160/

File: internal/series/series.go [hunk 2]
```diff
@@ -52,6 +52,7 @@ var Series = map[string]SeriesInfo{
 	"yakkety": {true, Ubuntu, true, 1.103},
 	"zesty":   {true, Ubuntu, true, 1.104},
 	"artful":  {true, Ubuntu, true, 1.105},
+	"bionic":  {true, Ubuntu, true, 0},
```
Comment (mhilton) [inline internal/series/series.go]: I'd suggest you should maybe put the boost number in a comment (I'd suggest 1.15)

**Review by mhilton — APPROVED** [review]
LGTM thanks

File: internal/charmstore/store.go (hunk 1 above)
Comment (rogpeppe) [reply internal/charmstore/store.go]: It was already yours :)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2161/

File: internal/series/series.go (hunk 2 above)
Comment (rogpeppe) [reply internal/series/series.go]: done

**Comment by rogpeppe** [comment]
$$merge$$

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2162/

**Comment by rogpeppe** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**